### ISSUE TITLE
urllib3 vulnerability

### ISSUE THREAD

**Issue by jbmoorhouse** [docker/docker-py#3181]
# Problem

`urllib3` version bump. Note the link below yields a `404`.

| Library | Vulnerability | Severity | Installed Version | Fixed Version | Title |
|--------|--------|--------|--------|--------|---------|
| urllib3 | CVE-2023-43804 | MEDIUM | 1.26.15 | 2.0.6, 1.26.17 | `Cookie` HTTP header isn't stripped on cross-origin redirects https://avd.aquasec.com/nvd/cve-2023-43804  |

# Anything Else?

It looks like this is being addressed in https://github.com/docker/docker-py/pull/3180

**Comment by mattelen** [comment]
Ah I'm also getting hit with this. A speedy result would be much appreciated.

**Comment by HubertBos** [comment]
Hi 

Any plans to resolve this issue shortly?

**Comment by akx** [comment]
Installing this library does not require exactly that version of `urllib3`:

https://github.com/docker/docker-py/blob/78439ebbe1aae77ff0fd9b666894d80807182e28/setup.py#L12-L17

There is nothing the maintainers of this library need to do so downstream projects and applications can use a newer version; `requirements.txt` is only used for tests.

**Comment by milas** [comment]
Yes, as mentioned above, you should be able to use newer, compatible versions of urllib3, the `setup.py` defines a minimum version.

Regardless, the version in `requirements.txt` (which is used for tests/development) has been bumped thanks to dependabot in #3183, so hopefully that will eliminate any noise from security scanners.
